Global/editUserModal.php

```diff
@@ -11,6 +12,13 @@
             </div>
             <form action="../User/userController.php" method="POST" onkeyup="checkUserDup()">
                 <div class="modal-body">
+                    <div>
+                        <label>Upload Image here</label>
+                    </div>
+                    <div class="custom-file" ID="uploadImageDiv">
+                        <input type="file" class="custom-file-input" id="customFile" onchange="validateImage()">
```

Comment:
Use https://mdbootstrap.com/docs/jquery/forms/file-input/
Having an upload button in the left side allows Javascript to handle the file to send to PHP, will make code much cleaner